automation/android/espresso/gradle.properties

```diff
@@ -0,0 +1,4 @@
+org.gradle.jvmargs=-Xmx2560M
+org.gradle.caching=true
+android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
Lines 3 and 4 did the jetifier thing.  I'm hoping that this works in bitrise as well as Android Studio.

Reply:
yep it should